src/redux/books/books.js

```diff
@@ -7,26 +7,37 @@
 /* eslint-disable no-unused-expressions */
```

Comment:
- [ ] It is not always a good idea to disable the linters. They are there to help apply standard and acceptable code. You have disabled 4 linter variables that can all be fixed easily. Give this a quick fix.